content/articles/how-to-create-horizontal-scrolling-menu-using-jquery-and-php/index.md

```diff
@@ -0,0 +1,216 @@
+Also, you need a web server installed. In our case, we will use XAMP(Apache HTTP Server), but you can choose to download your own choice, such as WAMP server, Nginx Web Server, Lighttpd Web Server, etc.
+
+### Significance of utilizing flat menu
+ * They limit time spent looking over what clients should perform to arrive at their ideal substance/content.
+ * Using horizontal scrolling menus saves vertical page space by setting a unique component.
+ * It keeps the menu thing list from wrapping to the accompanying line, which will be acceptable while thinking about vertical dividing and style.
+ * The best strategy for showing a rundown menu of things in a solitary way without using a scrollbar.
+ * It will be reasonable to make a responsive menu interface viable with web and versatile ports.
+
+### Available modules and libraries to make navigation menus
+There are a lot of available libraries and modules out on the web; nonetheless, in this article, we will utilize Jquery with the blend of PHP and HTML to make our menu bar that intermediate and beginner web developers will understand.
+
+### Advantages of exploitation custom code to execute flat looking over
+1. Own customization will consistently work on the code and limit the quantity of documents size.
+1. Contrasted with the devoted third-party modules, the custom code stream will be straightforward.
+1. The menus are adjusted in focus with the utilization of CSS text-align highlight
+
+### Concerning Menu model
+Our example code will be straightforward and fundamental to make a unique parchment menu to create our unique model. JQuery techniques are utilized to deal with flat scrolling and movement.
+On the snap occasion of these controls, the jQuery vivify() technique is called to move the menu things to and fro.
+
+### Code execution to call attention to Menu with flat scrolling
+Navigation symbols :
+ Rightward squiggle & leftward squiggle arrows will be employed in our tutorial.
+We will employ the above symbols to maneuver over the menu, but you can always use any other direction symbols of your choice. With a click, each opposite arrow will help in scrolling over the menu to and fro, thus assisting visitors in fetching menu items quickly on the page.
+
+
+### Hypertext markup language to show menu and result
+This html code includes right and left-way route symbols to assist clients in accessing the information base diverse menu items. The menu items are taken from the table information in our menu2 database.A simple navigation menu with four divs will be used in our HTML code. The first two divs include our user interface arrows, labeled &#8669 and &#8668, respectively.
+
+We'll use their Ids to style the UI arrows in CSS. The third and fourth divs will let us work with and decorate our database table items. The PHP foreach construct allows us to show and iterate over the contents of our SQL database in our menu model.

```

Comment:
When you say 'this html code', which HTML code does it refer to?, you have codes for HTML symbols, what are they doing? are your readers able to translate them?

Reply:
solved